Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
How about fixing somewhere 2.0 preview?
I think that it is desirable to have sample code of dotnet 2.0 running with docker.